### DISCUSSION TITLE
Using airodump-ng instead of hcxdumptool with hcxpcapngtool

### DISCUSSION THREAD

**Discussion by riverfinch** [ZerBea/hcxdumptool#252] [Q&A] [answered]
I tried using hcxdumptool but as mentioned in the issue https://github.com/ZerBea/hcxdumptool/issues/251 I can't do it.

Therefore, I thought about using other tools to implement this attack.

Can I just collect packets with airodump-ng and then use hcxpcapngtool to check if the PMKID is captured?

**Comment by ZerBea** [comment, marked as answer]
hcxpcapngtool convert every cap/pcap/pcapng file.
Do not use wpaclean, because that will remove important frames from the dump file.

BTW:
I prefer tshark or Wireshark

**Reply by riverfinch** [reply]
Thanks for your answer.
So, the PMKID attack (https://hashcat.net/forum/thread-7717.html) can be implemented using airodump-ng/tshark etc.
The only difference is that we don't get an explicit result that PMKID is caught, and more unnecessary frames for this attack will be written to the cap file. I'm right?

**Comment by ZerBea** [comment]
Correct.Wireshark, tshark and airodump-ng are passive tools. They are not able to run an interactive PMKID attack.
